python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -2264,6 +2264,25 @@ keys:
                   valid_values:
                     - microseconds
                     - milliseconds
+            min_delay_dynamic:
```

Comment:
Please mention that min_delay_static and min_delay_dynamic are mutually exclusive and min_delay_static will take precedence.

Reply:
Addressed. Thanks!